Session.js

```diff
@@ -0,0 +1,31 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the client side and be used as a buffer  
+// for client to server information needed for other components of Virtual Movie Night.
+ 
+    constructor() {
```

Comment:
I think it would be cool if like the assigning of the variables was done automatically in the constructor instead of having to change them later. The constructor could expect an obj that looks like this 
const expected = {
  sessionId: 'EEEE7',
  screenNameOfController: 'chris',
  ipOfVM: '1'
  listOfAttendees: ['chris', 'bryan']
};

and then simply this.sessionId_ = expected.sessionId

Reply:
wait sorry if my original comment confused you, I meant that you could pass in an object that would look like expected in the constructor, like constructor(obj) { ...setting various fields }, but a static method makes more sense to do this in

Reply:
no no you can get rid of the expected stuff, I was saying to do this in the fromObj like you did